nhais/Dockerfile.dynamodb

```diff
@@ -0,0 +1,24 @@
+FROM amazon/dynamodb-local
+
+USER root
+
+RUN yum install --assumeyes --quiet python3 && pip3 install -q awscli
+
+USER dynamodblocal
+
+# Create a table called mhs_state with primary key "key"
+RUN nohup bash -c "java -jar DynamoDBLocal.jar -sharedDb &" && \
+         AWS_ACCESS_KEY_ID=test AWS_SECRET_ACCESS_KEY=test aws dynamodb \
+        --endpoint-url http://localhost:8000 \
+        --region eu-west-2 create-table \
+        --table-name transaction_id_counter \
```

Comment:
1) This table should be renamed to 'generated_id_conter'
2) I would need to add a nhais_state table in the task NIAD-155. Should I create a similar file and then we will merge it, or should I just prepare code that will have placeholder for the table and add it after you'll merge this pr, or should I create branch from this branch to have your changes?

Reply:
I do still need to remove this table from the MHS dockerfile!